srcpkgs/termdown/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'termdown'
+pkgname=termdown
+version=1.16.0
+revision=1
+archs=noarch
+build_style=python3-module
+pycompile_module="termdown"
+hostmakedepends="python3-setuptools"
+depends="python3-setuptools python3-dateutil python3-click python3-pyfiglet"
+checkdepends=$depends
```

Comment:
Nice, caring of tests is quite rare. But please brace and quote.

Reply:
I've actually added test in the recent changes. Is it OK to use screen to test apps, that requires terminal capabilities?

Reply:
Well, `screen false` returns success, so it is not useful. Default do_check for that package has 0 tests, but it imports module, what is non-negligible test in as dynamic language as python is.